api/renter_test.go

```diff
@@ -66,4 +67,5 @@ func TestUploadAndDownload(t *testing.T) {
 	if upRoot != downRoot {
 		t.Error("uploaded and downloaded file have a hash mismatch")
 	}
+	os.Remove(downloadName)
```

Comment:
I'd rather leave the file. If the renter has been moved to the new folder paradigm then it'll go in a temp folder instead of cluttering up the code folder.